crates/solidity/inputs/language/src/definition.rs

```diff
@@ -2603,26 +2603,19 @@ codegen_language_macros::compile!(Language(
                     items = [
                         Struct(
                             name = EventDefinition,
-                            error_recovery = FieldsErrorRecovery(terminator = semicolon),
-                            fields = (
-                                event_keyword = Required(Terminal([EventKeyword])),
-                                name = Required(Terminal([Identifier])),
-                                parameters =
-                                    Optional(kind = NonTerminal(EventParametersDeclaration)),
-                                anonymous_keyword = Optional(kind = Terminal([AnonymousKeyword])),
-                                semicolon = Required(Terminal([Semicolon]))
-                            )
-                        ),
-                        Struct(
-                            name = EventParametersDeclaration,
                             error_recovery = FieldsErrorRecovery(
+                                terminator = semicolon,
```

Comment:
I suggest keeping the separate `EventParametersDeclaration` node, even if required, to be consistent with other parameter declarations in the grammar.

Reply:
This is only introduced in v2 and wrongly so, since it allows to skip skip parameters with parentheses altogether in the event definition. This brings v2 in line with v0/v1 and is one of the reasons for this PR to fix this definition.

Reply:
I meant keeping the node itself, even if it is now required. This is similar to `ParametersDeclaration`, `YulParametersDeclaration`, `ErrorParametersDeclaration`, etc...

Reply:
Done in b716ed00617fc6140beb2ae7caa64f7180487cbc